GameProject/Assets/TurnBasedAssets/Scripts/PlayerControls/PlayerController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using TurnBasedAssets.Scripts.Controllers;
+using UnityEngine;
+
+namespace TurnBasedAssets.Scripts.PlayerControls
+{
+    public class PlayerController : Controller
+    {
+        private IEnumerable<Vector3> _path = new List<Vector3>();
+        private List<GameObject> _pathVisualized = new List<GameObject>();
+        private Vector3 _previousLocation;
+        private Vector3 _previousDistance;
+        private float SphereRadius => mouseSelectionScript.MovableRadius;
```

Comment:
That's a clever use of a property to essentially alias that value though private properties are not the norm. No need to refactor this, just leaving a remark.

Reply:
Thank you